step.yml

```diff
@@ -76,7 +76,7 @@ inputs:
     - "true"
     - "false"
     is_dont_change_value: true
-- verbose: false
+- verbose: "false"
```

Comment:
Can you tell me which rule required this to put in `""`? I can see another bool value above it not requiring the same change.

Reply:
Step inputs are `string -> string` dicts of `input_key -> default_value` and this is now enforced in the schema. `is_dont_change_value` is not an input, it's a boolean option of an input.